src/app/routes/onDemandAudio/tempData/podcastExternalLinks/index.js

```diff
@@ -0,0 +1,49 @@
+import pathOr from 'ramda/src/pathOr';
+
+import nodeLogger from '#lib/logger.node';
+import { PODCAST_SERVICE_MISSING } from '#lib/logger.const';
+
+const logger = nodeLogger(__filename);
+
+const podcastExternalLinks = {
+  arabic: () => import('./arabic.js'),
```

Comment:
can we definitely use non-loadable dynamic imports on the client without a polyfill?

Reply:
I have not seen anywhere that non-loadable dynamic imports are an issue. They seem to be managed well by webpack ini our current config. However, since we don't use clieint-side routing for podcast pages yet, it won't be an issue now.

Reply:
Yes we can definitely use js dynamic imports on client with Webpack's polyfill